matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/Session.kt

```diff
@@ -330,4 +331,9 @@ interface Session {
      * Debug API, return the list of all RealmConfiguration used by this session.
      */
     fun getRealmConfigurations(): List<RealmConfiguration>
+
+    /**
+     * Returns the MultiRoomService associated with the session.
+     */
+    fun multiRoomLocationsService(): MultiRoomService
```

Comment:
@valerianglobekeeper shouldn't it be `multiRoomService()` without `Locations` as it will include other multi room messages later such as `panic`

Reply:
agreed